social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOC_People_Activity_CommentTestIT.java

```diff
@@ -200,6 +200,7 @@ public void test02_CommentOnYourFriendsActivity() {
     $(byId("cke_CommentTextarea" + id)).waitUntil(Condition.appears, Configuration.timeout).click();
     executeJavaScript("CKEDITOR.instances.CommentTextarea" + id + ".insertText(\"" + comment + "\")", "");
     // click on the button comment
+    Utils.pause(2000);
```

Comment:
`Utils.pause` does nothing now, the content of this method has been removed, it was the first step before remove it from all the source code. As we had a lot of refactoring, I decided to not remove it at first step but now, as the refactoring process is now mastered,  we can remove it. I will create a PR for that.

So this line of code can't have any impact on the UI test result.